examples/meta/src/classifier/shareboost.sg

```diff
@@ -0,0 +1,30 @@
+CSVFile f_feats_train("../../data/classifier_4class_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_4class_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_4class_2d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/classifier_4class_2d_linear_labels_test.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+MulticlassLabels labels_train(f_labels_train)
+MulticlassLabels labels_test(f_labels_test)
+#![create_features]
+
+#![create_instance]
+ShareBoost shareboost(features_train, labels_train, 30)
+#![create_instance]
+
+#![train_and_apply]
+shareboost.train()
+RealSubsetFeatures features_test_sub(features_test, shareboost.get_activeset())
```

Comment:
you need to add it to the cpp json, `generator/targets/cpp.json` in the same style as RealFeatures etc